### ISSUE TITLE
[autocomplete] Missing aria-expanded Required ARIA attribute is missing

### ISSUE THREAD

**Issue by MichelGratton** [element-plus/element-plus#12517]
Bug Type: **`Other`**

## Environment

- Vue Version: `3.2.47`
- Element Plus Version: `2.3.3`
- Browser / OS: `Chrome 112.0.5615.49 / Mac OS 13.2.1`
- Build Tool: `Vite`

### Related Component

- `el-autocomplete`

### Reproduction Link

[Docs](https://element-plus.org/en-US/component/autocomplete.html#basic-usage)

### Steps to reproduce

With a WCAG validator like Site Improve, or in [WAI-ARIA documentation](https://www.w3.org/TR/wai-aria-1.1/#combobox) or in [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role)

### What is Expected?

To be compliant with level A success criterions 1.3.1 (Info and Relationships) and 4.1.2 (Name, Role, Value) the element with `role="combobox"` must have an attribute `aria-expanded` in the auto-complete component, this element is the `div.el-autocomplete.el-tooltip__trigger`.

`aria-expanded` must be set to **false** if the popper list is hidden and **true** if the popper list visible.

See [WAI-ARIA documentation](https://www.w3.org/TR/wai-aria-1.1/#combobox) or in [Mozilla documentation](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role)

### What is actually happening?

The attribute `aria-expanded` is missing and do not comply with success criterions 1.3.1 and 4.1.2.

Required ARIA attribute is missing

This element has a role that should be accompanied by specific ARIA attributes.
role="combobox"

missing aria-expanded

### Additional comments

_(empty)_

![image](https://user-images.githubusercontent.com/12684802/233106512-6f97cfc3-75ed-48ee-911d-6c55bfe7b01c.png)
